nncf/compression_method_api.py

```diff
@@ -181,7 +182,7 @@ def export_model(self,
                               opset_version=10,
                               training=True)  # Do not fuse Conv+BN in ONNX. May cause dropout nodes to appear in ONNX
         model.forward = original_forward
-
+        model.load_state_dict(original_state_dict)
```

Comment:
Don't forget to remove this if it really is not necessary.

Reply:
Done